content/docs/2.4/scalers/nats-streaming.md

```diff
@@ -32,7 +32,9 @@ triggers:
 
 ### Authentication Parameters
 
-Not supported yet.
+**Connection Authentication:**
```

Comment:
Can you move this to v2.5 and align with the [scaler documentation guidance](https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler) please?

For the rest it looks OK.

Reply:
oh right . yes sure @tomkerkhove